src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -115,8 +116,7 @@ class IndexLogManagerImpl(indexPath: Path) extends IndexLogManager {
       FileUtil.copy(fs, pathFromId(id), fs, latestStablePath, false, new Configuration)
     } catch {
       case ex: Exception =>
-        // TODO: replace printStackTrace with Logging.
-        ex.printStackTrace()
+        logError("", ex)
```

Comment:
Can we add a message here? (e.g., `s"Failed to create the latest stable log with id = '$id'"`.

Reply:
Updated. Thanks!